src/bootstrap/build/config.rs

```diff
@@ -67,6 +67,7 @@ pub struct Config {
     pub target: Vec<String>,
     pub rustc: Option<String>,
     pub cargo: Option<String>,
+    pub rebuild: bool,
```

Comment:
Could you rename this to `local_rebuild`? We may actually use a key like "rebuild" to indicate whether dependencies should be tracked eventually as well.